boudicca.base/remote-collector/remote-collector-api/src/main/kotlin/base/boudicca/api/remotecollector/RemoteCollectorApi.kt

```diff
@@ -0,0 +1,28 @@
+package base.boudicca.api.remotecollector
+
+import base.boudicca.api.remotecollector.model.EventCollection
+import io.swagger.v3.oas.annotations.OpenAPIDefinition
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import javax.ws.rs.GET
+import javax.ws.rs.POST
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+
+@OpenAPIDefinition
+interface RemoteCollectorApi {
```

Comment:
So i guess we do not see this api as on the same level as the publisher api?

Reply:
why do think this way? because i moved it into a subfolder? that was only done to group it better and we already have a looot of projects in the base.boudicca folder

Reply:
Just because it was in a subfolder and hence different from the other apis. but grouping makes sense to me.